frontend/src/components/home/Apply.js

```diff
@@ -1,7 +1,67 @@
 import React from 'react'
+import { Typography, Box, Grid, Button } from '@mui/material'
+
 
 function Apply() {
-    return <div>-apply component placeholder for home page-</div>
+    return <div>
+        <Box sx={{ padding: "50px", }}>
+            <Typography variant='h2' sx={{ fontFamily: "Lato", fontWeight: "medium", textAlign: "center" }}>
+                Get Involved
```

Comment:
we can decrease the size of the font and the color should be dark blue instead of black (access it through theme)